test/generate-vdm.spec.ts

```diff
@@ -76,7 +76,7 @@ describe('generate-vdm', () => {
   });
 
   it('should resolve the projectdir', async () => {
-    const projectDir = '/root';
+    const projectDir = `${path.sep}root`;
```

Comment:
Please check how the other tests create folders for their output and use that instead.